todoman/cli.py

```diff
@@ -80,6 +80,14 @@ def _validate_start_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_category(ctx, param, val):
+    ctx = ctx.find_object(AppContext)
+    try:
+        return ctx.formatter.parse_category(val)
+    except ValueError as e:
```

Comment:
I don't see how this helps catch that very-specific bug. If a `ValueError` is raised, or a `BadParameter` is raised, we're still not catching that possible bug, because no tests will cover that new scenario.